samples/chat/csharp-ooproc/ChatBotIsolated.cs

```diff
@@ -79,7 +78,7 @@ public class CreateChatBotOutput
 
     public class PostResponseOutput
     {
-        [ChatBotPostOutput("{chatId}", Model = "gpt-3.5-turbo")]
+        [ChatBotPostOutput("{chatId}", Model = "%AZURE_DEPLOYMENT_NAME%")]
```

Comment:
Is this change necessary? If so, do we need to make any corresponding changes to the README to account for this new environment variable requirement when people run the sample locally?

Reply:
Yeah we do need this variable for running the E2E test. This is similar to how the in-proc Chat completion E2E tests were running, in which we need to specify the azure deployment name. I'll update the read me with this info!

Reply:
Yeah I do think we need this change since the E2E tests rely on the azure deployment name within the model, similar to how the E2E tests were working the csharp in-proc model. I went ahead and updated the README